src/platform/cert_capi_openssl.c

```diff
@@ -87,15 +87,12 @@ CxPlatTlsVerifyCertificate(
         CertFlags |= CERT_CHAIN_REVOCATION_CHECK_CHAIN_EXCLUDE_ROOT;
     }
 
-    uint32_t IgnoreFlags =
```

Comment:
I feel like I did this on purpose when I wrote it, but I don't remember now, so I'm removing it.